mautic/mautic
# Update .gitpod.Dockerfile

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

DDEV is out of date in our Gitpod config file, preventing it from loading:

![screenshot-mautic-mautic-ixsuuxr1t5v ws-eu115 gitpod io-2024 07 12-17_39_39](https://github.com/user-attachments/assets/7f0dcd3d-b126-4627-865c-295d44b62cce)

---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure DDEV loads and the Mautic instance spins up. You may need to use `ddev restart` if the web/db containers are slow to load.
